tools/cross_runner/src/gemm.h

```diff
@@ -19,8 +19,11 @@ enum class GemmType
     // q + kv
     GemmType_QK_Q_KV,
     GemmType_SV_S_KV,
+    //dpas
+    GemmType_QK_QKV_DPAS,
```

Comment:
> Instead of new GemmType type I recommend to add cmd argument `"--use_dpas"` to GemmCmDispatcher::cm_params_t class. This way we will avoid a lot of if conditions like for example `else if (type_ == GemmType::GemmType_QK_QKV || type_ == GemmType::GemmType_QK_QKV_DPAS )` and we can use have very little if conditions with `"--use_dpas"` in kernel/build options selection mechanism

Based on your suggestion, I may need to change a logic for selecting CM shader. Could you help check whether it may work with the change on the right side.

<img width="1393" alt="image" src="https://github.com/smarcink/dml_runner/assets/77707697/923c3fde-4e3b-416d-b58c-6a5c960d0b1f">

Reply:
Yes, that looks good!